steps/start/benefit-type/BenefitType.js

```diff
@@ -51,15 +47,19 @@ class BenefitType extends SaveToDraftStore {
     }
 
     const allowedTypes = [benefitTypes.personalIndependencePayment];
-    if (allowESA) {
+    if (isFeatureFlagEnabled('allowESA')) {
```

Comment:
As ESA benefit switched on now, can we not just remove this?

Reply:
Done